ssn/integrated/ssnx.ttl

```diff
@@ -54,7 +54,30 @@ time:TemporalEntity a owl:Class .
   owl:imports sosa: , ssn: .
 
 
+## Features of interest and observable property
 
+oldssn:FeatureOfInterest a owl:Class ;
+  rdfs:label "Feature of Interest"@en ;
+  skos:definition """A feature is an abstraction of real world phenomena (thing, person, event, etc)."""@en ;
+  dcterms:source """skos:exactMatch 'feature' [O&M]
+    http://www.opengeospatial.org/standards/om""" ;
+  rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Skeleton#Skeleton" ;
+  owl:deprecated true ;
+  owl:equivalentClass sosa:FeatureOfInterest ;
+  rdfs:isDefinedBy <http://purl.oclc.org/NET/ssnx/ssn> .
+
+oldssn:Property a owl:Class ;
+  rdfs:label "Property"@en ;
+  skos:definition """An observable Quality of an Event or Object.  That is, not a Quality of an abstract entity, but rather an aspect of an entity that is intrinsic to and cannot exist without the entity and that is observable by a sensor."""@en ;
+  dcterms:source """skos:exactMatch 'property' [O&M]
+             http://www.opengeospatial.org/standards/om""" ;
+  rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Skeleton#Skeleton" ;
+  owl:deprecated true ;
+  owl:equivalentClass ssn:Property ;
+  rdfs:isDefinedBy <http://purl.oclc.org/NET/ssnx/ssn> .
+  
```

Comment:
Implementation of the option 6 in https://www.w3.org/2015/spatial/wiki/Property_and_Observable_Property